index.css

```diff
@@ -504,3 +567,370 @@ textarea {
+    margin-right: 110px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 15px;
+    line-height: 20px;
+  }
+
+  .roger {
+    height: 870px;
+    display: flex;
+    padding: 0 0 0 37px;
+    background: #1c1a19 url(Desktop/muñeco2.png) 100% 125% no-repeat;
+  }
+
+  .pad {
+    width: 566px;
+    height: 160px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-size: 50px;
+    font-weight: 400;
+    line-height: 82px;
+    margin-top: 130px;
+    margin-left: 80px;
+    color: #f55800;
+  }
+
+  .supporttest {
+    color: #fff;
+    width: 566px;
+    height: 24px;
+    margin: 20px 0 0 90px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 32px;
+    line-height: 44px;
+  }
+
+  .developer {
+    margin: 30px 0 0 0;
+  }
+
+  .theme1 {
+    color: #f4f5f7;
+    width: 440px;
+    height: 123px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 20px;
+    line-height: 32px;
+  }
+
+  .palet {
+    display: flex;
+    flex-direction: column;
+    justify-content: space-around;
+    align-items: center;
+    list-style: none;
+  }
+
+  .palet a {
+    position: relative;
+    bottom: 270px;
+  }
+
+  .palet li {
+    height: 45px;
+  }
+
+  .myresent {
+    display: flex;
+    margin-left: 120px;
+    background-image: url(../Desktop/line3.png);
+    background-repeat: no-repeat;
+    background-position: 80% 75%;
+    background-size: 63%;
+  }
+
+  footer h2 {
+    width: 450px;
+    height: 256px;
+    font-style: normal;
+    font-family: "Inter", sans-serif;
+    text-align: start;
+    font-size: 40px;
+    font-weight: 700;
+    line-height: 52px;
+    color: #172b4d;
+    margin: 0  0 0 150px;
+  }
+
+  .myresent h2 {
+    height: 52px;
+    font-family: "Crete Round", serif;
+    font-style: normal;
+    font-weight: 400;
+    font-size: 40px;
+    line-height: 52px;
+    margin-top: 110px;
+    background-image: none;
+  }
+
+  .box1 {
+    grid-column: 3;
+    grid-column-start: 3;
+    grid-column-end: 4;
+    margin-right: 10px;
+  }
+
+  .box1 img {
+    display: none;
+  }
+
+  .box1 .daily {
+    width: 410px;
+    height: 140px;
+    font-family: "Inter", sans-serif;
+    font-style: normal;
+    font-weight: 350;
+    font-size: 19px;
+    line-height: 26px;
+  }
+
+  .box2 {
+    height: 435px;
+    background: url(../Desktop/Placeholder.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box3 {
+    height: 435px;
+    background: url(../Desktop/Placeholder4.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box4 {
+    height: 435px;
+    background: url(../Desktop/Placeholder5.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
+  }
+
+  .box {
+    height: 435px;
+    background: url(../Desktop/Placeholder3.png);
+    background-repeat: no-repeat;
+    margin-top: 90px;
```

Comment:
- [ ] Please correct all URL paths for your background image by removing one `.` from the beginning of the path because now it's not working (appearing) online.
Hint: for example make it like that `background: url(./Desktop/Placeholder3.png);`

Reply:
thank you for your CR